### PR TITLE
Issue#250_Fix_exception_message_in_polls

### PR DESCRIPTION
**Pull request by kizimov**
dev
## ZenHub

* [Main ZenHub ticket](https://app.zenhub.com/workspaces/home-project-5f7b77ff8db73a001cb17009)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Comment by kizimov** [comment]
@Disabled("Associated with Issue# 250. Will be fixed in issue # 290.")
CooperationContactApiIT.passNullUserIdWhenDeleteAnyContactTest
CooperationContactApiIT.passNullCooperationIdWhenGetContactTest
CooperationContactApiIT.passNullIdWhenDeleteAnyContactTest
UserApiIT.passNullWhenDeleteUserTest
NewsApiIT.passNullWhenDeleteNewsTest
CooperationContactApiIT.passNullIdWhenDeleteAnyContactTest


@Disabled("NotValid test. Will be fixed in issue # 290.")
ContactApiIT.passNullWhenGetContactTest (124 in Api.mustache)
UserApiIT.passNullWhenReceivingTest
NewsApiIT.passNullWhenReceivingNewsTest

@Disabled("error similar to Issue#250. Will be fixed in issue # 290.")
UserApiIT.createNullUserTest

@Disabled("broken test disabled")
QueryHousePollIT.getAllHousesFromPollsByQuantityFlat
QueryPoll. getAllPollsDescSort

**Review by Illia-Shchur — APPROVED** [review]

File: home-api-tests/pom.xml [hunk 1]
```diff
@@ -401,5 +401,16 @@
             <artifactId>commons-text</artifactId>
             <version>${commons-text-version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
```
Comment (kh0ma) [inline home-api-tests/pom.xml]: Isn't it redundant?

File: home-api-tests/pom.xml [hunk 2]
```diff
@@ -401,5 +401,16 @@
...
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.jetbrains</groupId>
+            <artifactId>annotations</artifactId>
```
Comment (kh0ma) [inline home-api-tests/pom.xml]: Do we really need it?

File: home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/contacts/ContactApiIT.java [hunk 3]
```diff
@@ -43,8 +43,26 @@
...
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static com.softserveinc.ita.homeproject.api.tests.utils.ApiClientUtil.BAD_REQUEST;
+import static com.softserveinc.ita.homeproject.api.tests.utils.ApiClientUtil.NOT_FOUND;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
```
Comment (kh0ma) [inline home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/contacts/ContactApiIT.java]: Aren't these imports unused?

File: home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/invitations/InvitationApiIT.java [hunk 4]
```diff
@@ -12,12 +12,14 @@
...
 import com.softserveinc.ita.homeproject.model.CreateCooperation;
 import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
 class InvitationApiIT {
 
     private final CooperationApi cooperationApi = new CooperationApi(ApiClientUtil.getClient());
 
+    @Disabled("NotValid test")
```
Comment (kh0ma) [inline home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/invitations/InvitationApiIT.java]: Please add more context why this test is not valid.

Comment (kh0ma) [reply home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/invitations/InvitationApiIT.java]: It looks like it should be removed

**Review by kh0ma — CHANGES_REQUESTED** [review]

File: home-api-tests/pom.xml (hunk 1 above)
Comment (kizimov) [reply home-api-tests/pom.xml]: You are right. This is already in lines 383-386.  But then I need to delete line  385 (<version>${assertj-version}</version>)

File: home-api-tests/pom.xml (hunk 2 above)
Comment (kizimov) [reply home-api-tests/pom.xml]: We don't need this.

File: home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/contacts/ContactApiIT.java (hunk 3 above)
Comment (kizimov) [reply home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/contacts/ContactApiIT.java]: Removed unused imports.

File: home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/invitations/InvitationApiIT.java (hunk 4 above)
Comment (kizimov) [reply home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/invitations/InvitationApiIT.java]: You're right. I delete it.

**Review by kh0ma — APPROVED** [review]

**Review by Vladyslav-Frolov — APPROVED** [review]

File: home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/polls/QueryPoll.java [hunk 5]
```diff
@@ -70,7 +70,6 @@ void getAllPollsAscSort() throws ApiException {
     void getAllPollsDescSort() throws ApiException {
 
         createPoll();
-        createPoll();
```
Comment (kh0ma) [inline home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/polls/QueryPoll.java]: Why was it removed?

**Review by IhorSamoshost — APPROVED** [review]

File: home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/polls/QueryPoll.java (hunk 5 above)
Comment (kizimov) [reply home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/polls/QueryPoll.java]: I decided that in the previous tests we are already creating a number of polls. And that's enough. Maybe I was wrong.

**Pull request merged**